scan/src/contexts/AccountContext.re

```diff
@@ -0,0 +1,36 @@
+type t = {
+  address: Address.t,
+  privKey: JsBuffer.t,
+};
+
+type a =
+  | Connect(string)
+  | Disconnect;
+
+let reducer = _ =>
+  fun
+  | Connect(mnemonic) => {
+      let bandchain = BandWeb3.network("http://localhost:8010", "bandchain");
```

Comment:
Let's do this globally up top.

Reply:
This http://localhost:8010 will need to be revisited #1035

Reply:
Sure